docs/design/IamUserInlinePolicy.md

```
@@ -0,0 +1,102 @@
+
+In the process, we check if the IAM policy has explicit `DENY`, otherwise, we look for explicit `ALLOW`.
+If we have an IAM policy and we didn’t have both, then it is an implicit `DENY` and it would result in an `AccessDenied` error.
+
+## Demo
+Use an account (either it’s the admin account or a newly created one from the CLI).  
+In S3: the account creates a bucket.  
+In IAM: the account creates a user, an access key, and puts a user inline policy.  
+Check the ability of the user to perform S3 operations according to the IAM policy
+
+1. Create the alias for the account: 
+  - `alias account-s3=’AWS_ACCESS_KEY=<access-key> AWS_SECRET_ACCESS_KEY=<secret-key> aws --no-verify-ssl --endpoint-url <endpoint-url-s3>`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)